interface-elements-for-web/articles/card-view.md

```diff
@@ -4,7 +4,7 @@ author: Svetlana Nikulina
 legacyId: 115295
 ---
 # Card View
-This section describes the capabilities provided by the Card View control, which represents data in cards, supports data editing, sorting, filtering, summary calculation, and many other features.
+This section describes the Card View control's capabilities. It represents data in cards, supports data editing, sorting, filtering, summary calculation, and many other features.
```

Comment:
"... and many other features." This is too vague.

Reply:
Modified.